src/components/CreateCoursePage/CreateCourseForm.jsx

```diff
@@ -119,17 +125,17 @@ class BaseCreateCourseForm extends React.Component {
             }
             required
           />
-          <Field
+          {currentValues && ENTITLEMENT_TRACKS.includes(currentValues.enrollmentTrack) && <Field
```

Comment:
Feels weird to go up several layers for the currentValues from fromValues etc, just to grab the value of the field next to you here. But I suppose that's correct.

Reply:
formValues is the way to get the **live** updates from the form. It definitely does take a couple layers to get here though. I wish it were simpler. https://redux-form.com/8.1.0/docs/api/selectors.md/